examples/bsp_ext.py

```diff
@@ -0,0 +1,67 @@
+import os
+from pathlib import Path
+try:
+    import ruamel.yaml
+except ImportError:
+    print("!! ruamel.yaml package is not installed. No BSP extension is added !!")
+    quit()
```

Comment:
`quit` should only be used within the python shell. In scripts `sys.exit(code)` should be used instead, 

BTW, won't be better to exit with an error in case of failed import?